packages/accordions/src/styled/timeline/StyledTimeline.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'timeline.timeline';
+
+/**
+ * 1. <ol> reset.
+ */
+export const StyledTimeline = styled.ol.attrs({
```

Comment:
I had originally used a `<ul>` and @jzempel suggested the `<ol>`. I think it makes sense because a timeline is chronological (thus ordered). The `<ol>` seems more suitable from a semantic perspective.

Reply:
I agree that an `<ol>` here makes sense